Docker Build

aijaz-baig opened the pull request:
> > Moved configurator docker build to /tmp directory
> > Issue noticed if setup_nfp.py was invoked from
> >    /usr/lib/python2.7/site-packages/gbpservice/.... as a
> >    recursive copy of gbpservice/ into iteself fails

aijaz-baig committed:
Docker Build

>> Moved configurator docker build to /tmp directory
>> Issue noticed if setup_nfp.py was invoked from
   /usr/lib/python2.7/site-packages/gbpservice/.... as a
   recursive copy of gbpservice/ into iteself fails

rajendramachani commented on gbpservice/contrib/nfp/tools/image_builder/disk_image_create.py:
```diff
@@ -35,7 +36,7 @@ def create_configurator_docker():
 
     # create a docker image
-    os.chdir(cur_dir)
+    os.chdir(docker_build_dir)
```
This makes us to invoke the disk image script from image_builder directory only right ?

rajendramachani replied:
Its not a problem now

aijaz-baig committed:
Docker Build

>> setting the docker_build_dir incase the script is not invoked
   from setup_nfp.py
